### PR TITLE
Chore es6

### PR DESCRIPTION
**Pull request by Cyebukayire**
#### What does this PR do?
Enables the ES6

#### Description of changes made
- Set up Babel
- Changed  `require('')` and `module.export ` to `import from` and  `export default`

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,7 +1,6 @@
...
-DB_TEST_USERNAME=
-DB_TEST_PASSWORD=
-DB_TEST_NAME=
\ No newline at end of file
+DB_USER=postgres
+DB_PASSWORD=bestmom
+DB_NAME=phantom
+DB_TEST_USERNAME=postgres
+DB_TEST_PASSWORD=bestmom
+DB_TEST_NAME=phantom
```
Comment (TuyizeeAnastase) [inline .env.example]: Does this look like your db credentials ? 
refer to this
```
DB_USER='enter_your_db_username',
DB_PASSWORD=`enter_your_db_password'
```

File: package.json [hunk 2]
```diff
@@ -4,9 +4,9 @@
   "description": "",
```
Comment (TuyizeeAnastase) [inline package.json]: add project description and name is not `swagger-api-library`

**Review by TuyizeeAnastase — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (Cyebukayire) [reply .env.example]: OMG:) Sure, I should change that

File: package.json (hunk 2 above)
Comment (Cyebukayire) [reply package.json]: Description updated

**Review by IrakozeLoraine — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (claranceliberi) [reply .env.example]: I think this will be modified by @Cyebukayire  on his end this PR has nothing to do with them.

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
Consider changing routes to es6 as well

File: package.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "swagger-api-library",
```
Comment (TuyizeeAnastase) [inline package.json]: This is not project name

**Review by TuyizeeAnastase — CHANGES_REQUESTED** [review]
Merge and solve conflicts

**Review by thepatrickniyo — APPROVED** [review]

**Review by TuyizeeAnastase — APPROVED** [review]

**Pull request merged**